systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeScramShaST.java

```diff
@@ -129,6 +129,59 @@ void testReceiveSimpleMessageTlsScramSha() throws Exception {
         assertThat(deleteConsumer(bridgeHost, bridgePort, groupId, name), is(true));
     }
 
+    @Test
+    void testScramShaAuthWithWeirdNamedUser() throws Exception {
+        String topicName = "topic" + rng.nextInt(Integer.MAX_VALUE);
+        String groupId = "my-group-" + rng.nextInt(Integer.MAX_VALUE);
+
+        // Create weird named user with . and more than 64 chars -> SCRAM-SHA
+        String weirdUserName = "jjglmahyijoambryleyxjjglmahy.ijoambryleyxjjglmahyijoambryleyxasd.asdasidioiqweioqiweooioqieioqieoqieooi";
+
+        // Create topic
+        KafkaTopicResource.topic(CLUSTER_NAME, topicName).done();
+        // Create user
+        KafkaUserResource.scramShaUser(CLUSTER_NAME, weirdUserName).done();
+
+        JsonObject config = new JsonObject();
+        config.put("name", weirdUserName);
```

Comment:
I don't think it makes sense using the `weirdUserName` for creating a consumer on the bridge, it should be a different test. You are testing the how a `KafkaUser` username can be weird not the name of the consumer on the bridge. Just use a different simpler name here.